src/main/java/ir/sahab/neor/kafka/reader/KafkaProtoParquetWriter.java

```diff
@@ -153,27 +135,29 @@ private KafkaProtoParquetWriter(Builder<T> builder) {
         Validate.isTrue(!defaultFs.isEmpty());
         targetDir = new Path(defaultFs, builder.targetDir);
 
-        // Consumer poll timeout must be less than max file duration
-        this.kafkaPollTimeoutMillis = Math.min(DEFAULT_KAFKA_POLL_TIMEOUT,
-                maxFileOpenDurationMillis > 0 ? maxFileOpenDurationMillis : Long.MAX_VALUE);
-
         MetricRegistry metricRegistry = builder.metricRegistry;
         if (metricRegistry != null) {
             metricRegistry.register("parquet.writer.written.records", totalWrittenRecords);
             metricRegistry.register("parquet.writer.flushed.records", totalFlushedRecords);
             metricRegistry.register("parquet.writer.written.bytes", totalWrittenBytes);
             metricRegistry.register("parquet.writer.flushed.bytes", totalFlushedBytes);
         }
+        smartCommitKafkaConsumer = new SmartCommitKafkaConsumer<>(kafkaConfig,
```

Comment:
It is better to have a new line beginning of each section of the code.

Reply:
done